File: first.cpp
#include <iostream>
using namespace std;
  
#define PI 3.141
  
int main(){
    float radius, area, circumference;
    cout << "Enter radius of circle\n";
    cin >> radius;
    // Area of Circle = PI x Radius X Radius
    area = PI*radius*radius;
    circumference = 2*PI*radius;
    cout << "The ciircumference of circle is: " << circumference;
    cout << "and the area is: " << area;
    
    return 0;
}

File: first.java
package helpme;

import java.util.Scanner;

public class first {
  public static void main(String[] args) {
    Scanner userInput = new Scanner(System.in);
    
    System.out.print("Enter the radius of your circle: ");
    
    int radius = userInput.nextInt();
    
    double circumference = 2 * Math.PI * radius;
    double area = Math.PI * (radius * radius);
    
    System.out.println("The circumference is: " + circumference + ", and the area is: " + area);
  }
}
